docs/source/azure/policy/resources/accesscontrol.rst

```diff
@@ -130,4 +136,57 @@ is Owner.
               op: eq
               value: custodian@example.com
          actions:
-            - type: delete
\ No newline at end of file
+            - type: delete
+
+Return all role assignments with the Subscription level scope access.
+
+.. code-block:: yaml
+
+    policies:
+       - name: assignments-subscription-scope
+         resource: azure.roleassignment
+         filters:
+            - type: scope
+              value: subscription
+
+Return all role assignments with the Resource Group level scope access.
```

Comment:
You only really need one example of using the scope filter alone since the schema shows you have the options subscription or resource-group.